packages/expo-screen-capture/ios/ScreenCaptureModule.swift

```diff
@@ -45,9 +46,12 @@ public final class ScreenCaptureModule: Module {
     if shouldListen && !isListening {
       // swiftlint:disable:next line_length
       NotificationCenter.default.addObserver(self, selector: #selector(self.listenForScreenCapture), name: UIApplication.userDidTakeScreenshotNotification, object: nil)
+      // swiftlint:disable:next line_length
+      NotificationCenter.default.addObserver(self, selector: #selector(self.listenForScreenRecording), name: UIScreen.capturedDidChangeNotification, object: nil)
```

Comment:
Silencing the errors is not the best idea (although I see the line above does it, you can fix it while you're at it 😄), we can do this:
```suggestion
    NotificationCenter.default.addObserver(
      self,
      selector: #selector(self.listenForScreenCapture),
      name: UIApplication.userDidTakeScreenshotNotification,
      object: nil
    )

    NotificationCenter.default.addObserver(
      self,
      selector: #selector(self.listenForScreenRecording),
      name: UIScreen.capturedDidChangeNotification,
      object: nil
    )
```